components/Head.js

```diff
@@ -15,6 +16,12 @@ export default function HeadComponent({
   if (!image.startsWith('http') && !image.startsWith('https')) image = `${url}${image}`
   title = `${title} | AsyncAPI Initiative`
 
+  //enable google analytics
```

Comment:
added this small comment as it took me some time to figure out where google analytics code is 😅